src/applications/personalization/components/IdentityNotVerified.jsx

```diff
@@ -15,6 +15,7 @@ const IdentityNotVerified = ({
     </p>
   ),
   additionalInfoClickHandler = null,
+  level = 3,
```

Comment:
We need to be able to customize the heading level of the AlertBox this component renders.